qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManageLocationAttributeTypesPage.java

```diff
@@ -95,7 +98,7 @@ public void clickOnCancelButton() {
 	}
 
 	public void purgeLocation() {
-		clickOn(DELETE_LOCATION_ICON);
+		clickOnLast(DELETE_LOCATION_ICON);
```

Comment:
Any reason why changing from `clickOn()` to `clickOnLast()`?

Reply:
the whole reason behind that is to be able to click on the right attribute to delete since in the `Location Attribute table`, a new row is created as the last

Reply:
Doesn't `clickOn()` do the work efficiently?